### PR TITLE
CSS for lunchbox MVP! plus modal skeleton

### PR DESCRIPTION
**Pull request by KHarlow03**
MVP for lunchbox styles prior to presentation, added skeleton for modal within HTML.

styles added:
shadow box for box items
granite countertop background
let's make lunch button styles
lunchbox handle is now centered

### CODE REVIEW

File: index.html
Comment (Bahashem) [inline index.html:1]: Change the spelling on line 65 congratulations. Otherwise ok.

**Review by Bahashem — CHANGES_REQUESTED** [review]
One change needed to HTML spelling but otherwise ok.

File: index.html
Comment (KHarlow03) [reply index.html:1]: done

**Review by Bahashem — APPROVED** [review]
Good to go.

**Pull request merged**